syft/workers/base.py

```diff
@@ -105,7 +108,7 @@ def _send_msg(self, message, location):
         raise NotImplementedError  # pragma: no cover
 
     @abstractmethod
-    def _recv_msg(self, message):
+    def _recv_msg(self, message: bin) -> bin:
```

Comment:
This method does not return binary data (it raises an exception). For specific implentions (ie virtual worker) it returns either a pointer tensor or None

Reply:
Ignore this comment